crates/tools/js/benchmark/index.js

```diff
@@ -380,6 +379,7 @@ function preprocessConfig(config) {
   }
 
   config.providerConfig.minGasPrice = BigInt(config.providerConfig.minGasPrice);
+  config.providerConfig.enableRip7212 = false;
```

Comment:
All other changes apart from this line were a result of the formatter. I needed to make this change to allow old snapshots to continue working with the newly added configuration field.

cc: @agostbiro